libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -125,6 +134,11 @@ public override async Task ContinueConversationAsync(string botAppId, Conversati
                 throw new ArgumentNullException(nameof(callback));
             }
 
+            if (_logger != null)
+            {
+                _logger.LogInformation($"Sending proactive message.  botAppId: {botAppId}");
+            }
```

Comment:
FYI this null `_logger` check pattern can be simplified and made less noisy using the null propagating operator like so:

```CSharp
_logger?.LogInformation($".....")
```

Reply:
great suggestion!